lib/log.test.js

```diff
@@ -0,0 +1,23 @@
+jest.mock('winston')
+const {createLogger} = require('winston')
+jest.unmock('winston')
+const {format, transports} = require('winston')
```

Comment:
remove this line. its not used anywhere.

Reply:
Without this line it throw error because `./log` uses those function which are from `'winston'` but it's already mocked.